catalog/gke/cluster/cluster.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: container.cnrm.cloud.google.com/v1beta1
+kind: ContainerCluster
+metadata:
+  name: example-us-east4 # {"$kpt-set":"cluster-name"}
+  namespace: config-control # {"$kpt-set":"platform-namespace"}
+  annotations:
+    cnrm.cloud.google.com/project-id: platform-project-id # {"$kpt-set":"platform-project-id"}
+    # Remove the default node pool after bootstrapping.
+    # Explcit node pool configuration allows for more isolation and makes it
+    # easier to replace node pools to change immutable fields.
+    cnrm.cloud.google.com/remove-default-node-pool: "true"
+  labels:
+    # Label with the environment to allow listing clusters by environment.
+    gke.io/environment: dev # {"$kpt-set":"environment"}
+spec:
+  location: us-east4 # {"$kpt-set":"location"}
+  # Default to the REGULAR channel.
+  # Use RAPID for faster access to features and fixes.
+  # Use STABLE for less disruption.
+  # Use UNSPECIFIED to unenroll from automatic updates.
+  releaseChannel:
+    channel: REGULAR
+  # Enable workload identity by default.
+  workloadIdentityConfig:
+    identityNamespace: platform-project-id.svc.id.goog # {"$kpt-set":"identity-namespace"}
+  addonsConfig:
+    # Enable Network Policy by default, for increased security.
+    # https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy
+    networkPolicyConfig:
+      disabled: false
+    # Enable NodeLocal DNSCache by default, for increased performance and scaling.
+    # https://cloud.google.com/kubernetes-engine/docs/how-to/nodelocal-dns-cache
+    dnsCacheConfig:
+      enabled: true
+    # Enable Compute Engine persistent disk CSI Driver by default, for access to
+    # volume snapshots and encryption with customer-managed encryption keys.
+    # https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/gce-pd-csi-driver
+    gcePersistentDiskCsiDriverConfig:
+      enabled: true
+  # Disable Pod Security Policy by default.
+  # Deprecated as of K8s v1.21
+  # https://github.com/kubernetes/kubernetes/pull/97171
+  podSecurityPolicyConfig:
+    enabled: false
```

Comment:
This is disabled by default right?

Reply:
The KCC docs do not specify:
https://cloud.google.com/config-connector/docs/reference/resource-docs/container/containercluster

Reply:
KCC probably wraps the TF provider, which also doesn't say, but it does call them Beta...
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#pod_security_policy_config

Reply:
I believe it should be disabled by default like other addon configs

Reply:
removed